CSharpBasics/Program.cs

```diff
@@ -223,4 +225,265 @@ public void Task_7()
+
+            SSD = 2;
+            priceSSD = 300;
+
+            GPU = 1;
+            priceGPU = 2500;
+
+            RAM = 4;
+            priceRAM = 200;
+
+            CPU = 1;
+            priceCPU = 950;
+
+            PowerSupplies = 1;
+            pricePowerSupplies = 300;
+
+            Motherboard = 1;
+            priceMotherboard = 400;
+
+            double totalCost = (SSD * priceSSD) + (GPU * priceGPU) + (RAM * priceRAM) + (CPU * priceCPU) + (PowerSupplies * pricePowerSupplies) + (Motherboard * priceMotherboard);
+            double discount = totalCost * 0.15;
+            double finalprice = totalCost - discount;
+
+            Console.WriteLine("Shopping List:");
+            Console.WriteLine($"SSD: {SSD} pcs. x {priceSSD} $.");
+            Console.WriteLine($"GPU: {GPU} pcs. x {priceGPU} $.");
+            Console.WriteLine($"RAM: {RAM} pcs. x {priceRAM} $.");
+            Console.WriteLine($"CPU: {CPU} pcs. x {priceCPU} $.");
+            Console.WriteLine($"PowerSupplies: {PowerSupplies} pcs. x {pricePowerSupplies} $.");
+            Console.WriteLine($"Motherboard: {Motherboard} pcs. x {priceMotherboard} $.");
+            Console.WriteLine("------------------------");
+            Console.WriteLine($"Total: {totalCost} $.");
+            Console.WriteLine($"Discount: {discount} $.");
+            Console.WriteLine($"Total amount due: {finalprice} $.");
+
+            Console.WriteLine("\nPress Enter to continue\n");
+            Console.ReadLine();
+        }
+
+        public void Task_6()
+        {
+            double a = 5;
+            double b = 7;
+
+            double c = (a++ + b / 2) + (a * b - a) / (a);
+
+            // (a++ + b / 2)
+            // b/2 = 7/2 = 3.5
+            // a+b/2 = 5+3.5 = 8.5 
+
+            // (a * b - a) / (a)
+            // (5 * 7 - 5) / (5) = 6
+            // 8.5 + 6 = 14.5
+
+            // (a++ + b / 2)
+            // 1) a++ = 5 , first use the value, then increase by 1
+            // 2) b / 2 = 7 / 2 = 3.5
+            // 3) a + b / 2 = 5 + 3.5 = 8.5
+
+            // (a * b - a) / a
+            // 1) a * b = 5 * 7 = 35
+            // 2) (5 * 7 - 5) / (5) = (35 - 5) / 5 = 6
+            // 3) 8.5 + 6 = 14.5
+
+            a = 5;
+
+            double d = (++a + b * b) / (a - b);
+
+            // (++a + b * b)
+            // 1) ++a = 5 + 1 = 6  
+            // 2) b * b = 7 * 7 = 49
+            // 3) ++a + b * b = 6 + 49 = 55
+
+            // (a - b)
+            // 1) a - b = 6 - 7 = -1
+
+            // (a - b) != 0, so division by (a - b) is valid
+
+            // (a + b * b) / (a - b)
+            // 1) (a + b * b) = 55
+            // 2) (a - b) = -1
+            // 3) (a + b * b) / (a - b) = 55 / -1 = -55
+
+            Console.WriteLine(c);
+            Console.WriteLine(d);
+
+            Console.WriteLine("\nPress Enter to continue\n");
+            Console.ReadLine();
+        }
+
+        public void Task_7()
+        {
```

Comment:
I love the fact that you created a nice styling to make program readable, but using everywhere ! to get opposite value is not the highest score :)